pkg/plugin/aggregation/aggregator.go

```diff
@@ -319,6 +319,7 @@ func (a *Aggregator) IngestResults(ctx context.Context, resultsCh <-chan *plugin
 
 		select {
 		case <-ctx.Done():
+			logrus.Debug("Stopping internal results processing")
```

Comment:
Removing this and the other logrus.Debug line.

A followup PR should make it possible to see these but right now we don't ever modify the level to view logrus lines at as far as I can tell.